src/App.js

```diff
@@ -1,25 +1,70 @@
-import logo from './logo.svg';
 import './App.css';
+import { useState } from 'react';
+import DisplayActivity from './Components/DisplayActivity';
+import Form from './Components/Form';
+import Error from './Components/Error';
 
 function App() {
+
+  const [activity, setActivity] = useState("");
+  const [activityLink, setActivityLink] = useState("");
+  const [formError, setFormError] = useState(false);
+
+  const getActivity = (numberOfKids, typeOfActivity) => {
+
+  console.log(numberOfKids)
+  console.log(typeOfActivity)
```

Comment:
Just getting started here. Super tiny nitpick but I've been called out on this before. Definitely remember to remove all console.log's before production. 😺